README.md

```diff
@@ -76,6 +76,7 @@ do not cover your particular topic, review their guides for guidance.
 - [Recommended Reading](reading.md)
 - [Responsibilities & Rituals](rituals/README.md)
 - [Tools](tools/README.md)
+- [Publishing gems with GitHub Packages](tools/gems.md)
```

Comment:
What do you think about putting this under the `ruby` folder?

Reply:
Done.